src/utils/types.ts

```diff
@@ -365,7 +365,8 @@ interface ResponseFull {
   optionIndex?: number
   reference?: any // Not yet decided how to represent
   isValid?: boolean | null
-  hash?: string
+  hash?: string // Used in Password plugin
+  files?: any[] // Used in FileUpload plugin
```

Comment:
hmm, this is a bit wired, i just noticed the ResponseFull having `hash` and `optionIndex` and `files` etc.. I thought as far as the 'core' code base is concerned, only thing that plugin updates is `value` and maybe `textValue` ? Otherwise they are not really `plugins` if they add types to core

Reply:
I think it's fine if we make a seperate issue to fix this for all of the plugins

Reply:
Yes I agree. So would you suggest that each plugin imports the basic "ResponseFull" and extends it with custom fields in their own file?

Reply:
I am not sure how to best deal with it atm, would be good to have a think about it next week, made an issue: https://github.com/openmsupply/application-manager-web-app/issues/456